pkg/spec/populators.go

```diff
@@ -163,7 +163,7 @@ func populateContainers(containers []Container, cms []ConfigMapMod, secrets []Se
 		// process envFrom field
 		c, err = populateEnvFrom(c, cms, secrets)
 		if err != nil {
-			return cnts, fmt.Errorf("error converting 'envFrom' to 'envs', app.containers[%d].%s", cn, err.Error())
+			return cnts, fmt.Errorf("error converting 'envFrom' to 'envs', deployment.containers[%d].%s", cn, err.Error())
```

Comment:
This will still be `app`, because of the following reasons:

- this is not tied to deployment in anyway
- user will still be considering it as app
- the error is user facing

Reply:
done